include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -124,11 +124,11 @@ class CommonOperatorLoader : public ProtobufLoader {
     auto *in0 = getOrCreateVariableByName(op.input(0));
     auto *in1 = getOrCreateVariableByName(op.input(1));
 
-    int broadcast = loadInt(dict["broadcast"]);
+    int broadcast = dict.count("broadcast") ? loadInt(dict["broadcast"]) : 1;
```

Comment:
Does this also depend on opset version?

Reply:
Yes.  I will add the version check.

Reply:
So seems like "broadcast" is one thing, that ONNX changed between versions 6 and 7. Maybe we can create virtual method `getBroadcast()` (or `loadBroadcast()`), which will be different for ONNX and Caffe2 importers. Caffe2's one will be trivial, and ONNX method will look at opset_version.